packages/apps/human-app/frontend/src/modules/worker/components/jobs-discovery/helpers/navigate-to-oracle-jobs.ts

```diff
@@ -0,0 +1,12 @@
+import { type NavigateFunction } from 'react-router-dom';
+import type { Oracle } from '@/modules/worker/services/oracles';
+import { routerPaths } from '@/router/router-paths';
+
+export const navigateToOracleJobs = (
```

Comment:
`handleHCaptchaNavigation` -> Both components perform very similar logic, but they have completely different names. Maybe we could use the same convention?